src/generic/animateg.cpp

```diff
@@ -327,7 +327,7 @@ wxSize wxAnimationCtrl::DoGetBestSize() const
     if (m_animation.IsOk() && !this->HasFlag(wxAC_NO_AUTORESIZE))
         return m_animation.GetSize();
 
-    return wxSize(100, 100);
+    return FromDIP(wxSize(100, 100));
```

Comment:
I also wonder if we shouldn't provide `FromDIP(x, y)` overload. It may make things a bit confusing, but writing `FromDIP(100, 100)` would be more palatable.

Or maybe add `SizeFromDIP()` or even just `SizeInDIP()`?

(this is just another note/thought without direct effect on this PR).